nestkernel/nestmodule.cpp

```diff
@@ -609,6 +609,17 @@ NestModule::CopyModel_l_l_DFunction::execute( SLIInterpreter* i ) const
   const std::string newmodname = getValue< std::string >( i->OStack.pick( 1 ) );
   DictionaryDatum dict = getValue< DictionaryDatum >( i->OStack.pick( 0 ) );
 
+  if ( oldmodname.toString() == "gap_junction" )
+  {
+    get_network().message( SLIInterpreter::M_WARNING,
```

Comment:
I think this should be an error, i.e., not a warning, and an exception should be raised. The user is asking NEST to do something that is not possible, therefore it is an error.